tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py26, py27, pypy
+envlist = py26, py27, pypy, py33, py34
```

Comment:
Ah, but with `u''` literals. That's probably good enough.

 - [ ] add pypy3 test target